eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/Schema.scala

```diff
@@ -12,15 +12,13 @@ import com.evolutiongaming.scassandra.TableName
  *  - [[SettingsCassandra]]
  *
  * @param journal stores all journals, see [[JournalStatements]]
- * @param metadata is not used any more, deprecated and scheduled for removal, removed in `14e143f178c1f9c1`
  * @param metaJournal stores information about active aggregates, see [[MetaJournalStatements]]
  * @param pointer is not used any more, deprecated and scheduled for removal, see [[PointerStatements]]
  * @param pointer2 stores information on how close to tail the replication is, see [[Pointer2Statements]]
  * @param setting used to track schema migrations, see [[SetupSchema]] and [[MigrateSchema]] for details
  */
 final case class Schema(
   journal: TableName,
-  metadata: TableName, // is not used // TODO MR remove with next major release
   metaJournal: TableName,
   pointer: TableName, // TODO MR remove with next major release
```

Comment:
What about this field?

Reply:
And table in general

Reply:
we wanted, but we may not rush till we have proof that nobody is relying on it any more